### PR TITLE
Fix #8846 UI: Replace all the modal with antd modal in the UI -4

### LINKED ISSUE #8846: UI: Replace all the modal with antd modal in the UI

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes :

I worked on the issue #8846 

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 1]
```diff
@@ -121,17 +121,6 @@
   border-radius: 6px;
 }
 
-// Text alignment
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/styles/app.less:124]: Why did we remove these stylings?

**Review by aniketkatkar97 — COMMENTED** [review]
One comment

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:124]: It was duplicate, that's why we have removed it.

**Review by aniketkatkar97 — DISMISSED** [review]

**Review by ShaileshParmar11 — CHANGES_REQUESTED** [review]
I left a few comments, Please remove the tailwind class wherever possible in your change

**Comment by Sachin-chaurasiya** [comment]
@ShaileshParmar11 , All comments are addressed, please review.

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**